MDCDecorator/MDCDecoratorEnd.yml

```diff
@@ -1,11 +1,10 @@
 steps:
 - task: Bash@3
-  displayName: MDC Extension End
+  displayName: DfD Post-Job Extension
   timeoutInMinutes: 1
   condition: always()
   continueOnError: true
   inputs:
     targetType: 'inline'
     script: | 
-      echo "Pipeline Start: $(pipelineStartTime)"

```

Comment:
How about adding a message to both decorators explaining that this was added automatically as part of MDC-DfD enablement.....

Reply:
Nice idea, I think we can add a description to the ADO task instead of printing out a message here. Let me look into it and add that right now

Reply:
Added a log group with a message